### PR TITLE
redbull: initial cr11 update

### PR DESCRIPTION
**Pull request by 306bobby-android**

### CODE REVIEW

File: barbet.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Google",
+			"device": "Pixel 5a",
+			"filename": "crDroidAndroid-15.0-20241117-barbet-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/barbet/11.x/crDroidAndroid-15.0-20241117-barbet-v11.0-BETA.zip/download",
+			"timestamp": 1731900723,
+			"md5": "b53375679a9fd8a5d0c2340ab8f7fcc9",
+			"sha256": "efcf4ab017e4bdc4a64028ddbfc0c1d4537848baa41976062ef56c23571c8141",
+			"size": 1366692811,
+			"version": "11.0",
+			"buildtype": "Monthly",
```
Comment (gwolf2u) [inline barbet.json]: BETA

File: barbet.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-official-13-14-15-crdroid-for-pixel-5a-barbet.4646786/",
```
Comment (gwolf2u) [inline barbet.json:15]: Update to new forum template

File: barbet_changelog.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Highlights & Device Specific Changes:
+Build type: Monthly / Stable
```
Comment (gwolf2u) [inline barbet_changelog.txt]: BETA

File: barbet_changelog.txt
Comment (gwolf2u) [inline barbet_changelog.txt:1]: also can update style of changelog to match others, but am ok like so also if really want to

File: bramble.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Google",
+			"device": "Pixel 4a",
+			"filename": "crDroidAndroid-15.0-20241117-bramble-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/bramble/11.x/crDroidAndroid-15.0-20241117-bramble-v11.0-BETA.zip/download",
+			"timestamp": 1731822357,
+			"md5": "d39ea752bc8a809fbc7e102d3741dec2",
+			"sha256": "de44f5ad48d349db784d0e887fc304e9283c1ac48bdb4e3abccdc0d641106c79",
+			"size": 1367672481,
+			"version": "11.0",
+			"buildtype": "Monthly",
```
Comment (gwolf2u) [inline bramble.json]: BETA

File: bramble.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-official-13-14-15-crdroid-for-pixel-4a5g-bramble.4664667/",
```
Comment (gwolf2u) [inline bramble.json:15]: update forum template

File: bramble_changelog.txt [hunk 6]
```diff
@@ -0,0 +1,18 @@
+Highlights & Device Specific Changes:
+Build type: Monthly / Stable
```
Comment (gwolf2u) [inline bramble_changelog.txt]: BETA

File: redfin.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Google",
+			"device": "Pixel 5",
+			"filename": "crDroidAndroid-15.0-20241117-redfin-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/redfin/11.x/crDroidAndroid-15.0-20241117-redfin-v11.0-BETA.zip/download",
+			"timestamp": 1731834538,
+			"md5": "4bf45d8599ac8096dc2392ee5d23ee42",
+			"sha256": "3b385cb41ca6414bb335c39c82901c38a621c84353390e5f1d2c1a97372e0c94",
+			"size": 1368014370,
+			"version": "11.0",
+			"buildtype": "Monthly",
```
Comment (gwolf2u) [inline redfin.json]: BEtA

File: redfin.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-official-13-14-15-crdroid-for-pixel-5-redfin.4664665/",
```
Comment (gwolf2u) [inline redfin.json:15]: update forum template

File: redfin_changelog.txt [hunk 9]
```diff
@@ -0,0 +1,16 @@
+Highlights & Device Specific Changes:
+Build type: Monthly / Stable
```
Comment (gwolf2u) [inline redfin_changelog.txt]: BETA

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by gwolf2u** [comment]
inactive

**Comment by 306bobby-android** [comment]
Active again, fixing right now. Template is merged on forum and will have updated files shortly with new build info and corrected build types (didn't realize monthly needed changed to beta!)

File: barbet_changelog.txt
Comment (306bobby-android) [reply barbet_changelog.txt:1]: Would it be ok if I wait and rewrite changelog to new format when beta is over?

Updated old to say BETA

Comment (gwolf2u) [reply barbet_changelog.txt:1]: ok with me
this status should match device.json and normal values are Testing/Alpha/Beta/Weekly/Monthly as per documentation
https://github.com/crdroidandroid/android_vendor_crDroidOTA/blob/14.0/README.md#12-changelogtxt-structure

**Review by gwolf2u — APPROVED** [review]

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**